vuepress/docs/next/docs/curate/bundle-component-details.md

```diff
@@ -0,0 +1,433 @@
+---
+sidebarDepth: 2
+---
+
+# Bundle Component Descriptors 
+
+
+Elements such as simple widgets, page templates, groups, and static resources are classified as platform components on Entando. This page describes each of these elements with an example.
```

Comment:
I might switch to "simple widgets, fragments, pages, content, and static resources". Pages and content are the most common types for sure so maybe even lead with them.